sys-devel/llvm/llvm-3.8.1-r1.ebuild

```diff
@@ -169,6 +169,12 @@ src_prepare() {
 	# https://bugs.gentoo.org/show_bug.cgi?id=578392
 	eapply "${FILESDIR}"/llvm-3.8-soversion.patch
 
+	# support building llvm against musl-libc
+	use elibc_musl && eapply "${FILESDIR}"/llvm-3.8-musl-fixes.patch
+
+	# support "musl" as a valid environment type in llvm
+	eapply "${FILESDIR}"/llvm-3.8-musl-support.patch
```

Comment:
That one patch is a little more intrusive than the others, so I felt it safer to apply it conditionally. I might be paranoid though.